src/Moose-SmalltalkImporter/CandidateListOperator.class.st

```diff
@@ -13,12 +13,15 @@ Class {
 
 { #category : #running }
 CandidateListOperator >> allCandidatesFor: invocation [ 
-	 
 	^Array 
 		withAll: 
 			(groups 
 				at: invocation signature 
-				ifAbsent: #())
+				ifAbsent: [ | stubs |
+					stubs := self importer ensureSmalltalkStubMethodsFor: invocation signature.
```

Comment:
Should this be here or in AbstractCandidateListOperator?

Reply:
There must be something in the #ifAbsent: and what was there before was not good.

There are other places were the whole solution could be improved, but currently it works and does not break anything.
We will make it better some other time (perfect is the enemy of good)

Reply:
OK, I hadn't understood the problem with putting it here or in the super class.
At first glance, I would say you are right and it should be in the super-class.
But I would have to spend more time on it to be sure.